app/config/api_filters.yml

```
@@ -50,6 +50,11 @@ services:
               url: 'exact'
         tags: [{ name: 'api_platform.filter', id: 'event.search' }]
 
+    resource.event.exists_filter:
+        parent: 'api_platform.doctrine.orm.exists_filter'
+        arguments: [ { feed: '~' } ]
+        tags: [{ name: 'api_platform.filter' }]
+
```

Consider reformatting `arguments` for consistency with the rest of the filter definitions:

```suggestion
    resource.event.exists_filter:
        parent: 'api_platform.doctrine.orm.exists_filter'
        arguments: 
            - feed: '~'
        tags: [{ name: 'api_platform.filter' }]

```